internal/data/receiver_verification.go

```diff
@@ -106,11 +135,17 @@ func (m ReceiverVerificationModel) Insert(ctx context.Context, sqlExec db.SQLExe
 		INSERT INTO receiver_verifications (
 		    receiver_id, 
 		    verification_field, 
-		    hashed_value
-		) VALUES ($1, $2, $3)
+		    hashed_value%s
+		) VALUES ($1, $2, $3%s)
 	`
 
-	_, err = sqlExec.ExecContext(ctx, query, verificationInsert.ReceiverID, verificationInsert.VerificationField, hashedValue)
+	if verificationInsert.UpdatedAt != nil {
+		query = fmt.Sprintf(query, ",\nupdated_at\n", ", $4")
+		_, err = sqlExec.ExecContext(ctx, query, verificationInsert.ReceiverID, verificationInsert.VerificationField, hashedValue, *verificationInsert.UpdatedAt)
+	} else {
+		query = fmt.Sprintf(query, "", "")
+		_, err = sqlExec.ExecContext(ctx, query, verificationInsert.ReceiverID, verificationInsert.VerificationField, hashedValue)
+	}
```

Comment:
What's the purpose of this change? We shouldn't be updating `updated_at` manually in the main code. 

If the only reason for this change is unit tests, we should find another way to do that. Either directly writing a SQL to update the updated_at field, or calling `UpdateReceiverVerification(..)` that will change the updated_at.

Reply:
ok I see what you're saying, I initially needed this change so that I could create fixtures "created at different times" so that I can test `GetLatestByPhoneNumber`